packages/utils/src/utils/validation.ts

```diff
@@ -74,12 +74,12 @@ export const validateLimitedAmount = (amount: string, decimals?: number, max?: s
   const value = safeParseUnits(amount, decimals)
 
   if (value !== undefined && value > BigInt(max)) {
-    return `Maximum value is ${safeFormatUnits(max, decimals)}`
+    return errorMsg || `Maximum value is ${safeFormatUnits(max, decimals)}`
   }
 }
 
-export const validateDecimalLength = (value: string, maxLen?: number, minLen = 1) => {
-  if (typeof maxLen === 'undefined' || !value.includes('.')) {
+export const validateDecimalLength = (value: string, maxLen?: number | null, minLen = 1) => {
+  if (maxLen == null || !value.includes('.')) {
```

Comment:
As above regarding undeclared `maxLen`.

Reply:
same as above. Those changes are in the validation.ts in production. I've just applied them to utils/validation.ts file.